src/applications/personalization/rated-disabilities/containers/RatedDisabilitiesApp.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import backendServices from 'platform/user/profile/constants/backendServices';
+import { fetchRatedDisabilities } from '../actions';
+
+// Wonder if we can put RD data in platform...
+import RequiredLoginView from 'platform/user/authorization/components/RequiredLoginView';
+import RatedDisabilityView from '../components/RatedDisabilityView';
+// import RatedDisabilityHeader from '../components/RatedDisabilityHeader';
+
+class RatedDisabilitiesApp extends React.Component {
+  render() {
+    const { ratedDisabilities } = this.props.ratedDisabilities;
+    return (
+      <>
+        {/* <RatedDisabilityHeader /> */}
+        <RequiredLoginView
+          authRequired={1}
+          serviceRequired={backendServices.USER_PROFILE}
+          user={this.props.user}
+          loginUrl={this.props.loginUrl}
+          verifyUrl={this.props.verifyUrl}
+        >
+          <RatedDisabilityView
+            fetchRatedDisabilities={this.props.fetchRatedDisabilities}
+            ratedDisabilities={ratedDisabilities}
+            user={this.props.user}
+          />
+        </RequiredLoginView>
+      </>
+    );
+  }
+}
+
+const mapStateToProps = state => ({
+  user: state.user,
+  ratedDisabilities: state.ratedDisabilities,
+});
```

Comment:
This can be done in a later ticket, but I think we'll need to add a prop here to make sure that the user is enrolled in the right required service before trying to render this app or hit the API. [Some examples in the Dashboard](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/personalization/dashboard/containers/DashboardApp.jsx#L359).